blueprints/02-at-scale/README.md

```diff
@@ -8,14 +8,23 @@ Once you have familiarized yourself with [CloudBees CI blueprint add-on: Get sta
-  | [Metrics Server](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/metrics-server/)               | This is a requirement for CloudBees CI HA/HS controllers for horizontal pod autoscaling.                                                                                                     |
-  | [Velero](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/velero/)                               | Backs up and restores Kubernetes resources and volume snapshots, which is only compatible with Amazon Elastic Block Store (Amazon EBS).                                                      |
+  | [AWS for Fluent Bit](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/aws-for-fluentbit/)| Acts as an applications log router for log observability in CloudWatch. |
+  | [Cluster Autoscaler](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/cluster-autoscaler/) | Watches Amazon EKS managed node groups to accomplish [CloudBees CI auto-scaling nodes on EKS](https://docs.cloudbees.com/docs/cloudbees-ci/latest/cloud-admin-guide/eks-auto-scaling-nodes). |
+  | [Kube Prometheus Stack](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/kube-prometheus-stack/) | Used for metrics observability.|
+  | [Metrics Server](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/metrics-server/) | This is a requirement for CloudBees CI HA/HS controllers for horizontal pod autoscaling.|
+  | [Velero](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/velero/)| Backs up and restores Kubernetes resources and volume snapshots, which is only compatible with Amazon Elastic Block Store (Amazon EBS).|
+  | [Bottlerocket Update Operator](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/addons/bottlerocket/)| It coordinates Bottlerocket updates on hosts in a cluster.|
+
+- [Amazon EKS blueprints Helm Release Add-on](https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/helm-release/) is used to install the following applications:
+
+| Helm Chart | Description |
+|-----------|--------------|
+| [Osixia Openldap](https://github.com/osixia/docker-openldap) | LDAP server. |
+| [AWS Node Termination Handler](https://github.com/osixia/docker-openldap) | Gracefully handle EC2 instance shutdown within Kubernetes (Addon is not compatible with Managed Instance Group see [issues#23](https://github.com/cloudbees/terraform-aws-cloudbees-ci-eks-addon/issues/23)). |
+| [Grafana Tempo](https://github.com/osixia/docker-openldap) | Tracing backend for [Jenkins OpenTelemetry](https://plugins.jenkins.io/opentelemetry/).  |
```

Comment:
- We need to add some extra space before each table row, so it remains indented within the bulleted list.
- I also added some minor copy edits and updated the link for the AWS Node Termination Handler, because it was using the same link as the Osixia Openldap repo. If the link I added is incorrect, please update.
```suggestion
  | Helm Chart | Description |
  |-------------------------------|-------------|
  | [Osixia Openldap](https://github.com/osixia/docker-openldap) | LDAP server. |
  | [AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler) | Gracefully handles EC2 instance shutdown within Kubernetes. Note that this add-on is not compatible with managed instance groups. For more information, refer to [issue #23](https://github.com/cloudbees/terraform-aws-cloudbees-ci-eks-addon/issues/23)). |
  | [Grafana Tempo](https://github.com/osixia/docker-openldap) | Provides backend tracing for [Jenkins OpenTelemetry](https://plugins.jenkins.io/opentelemetry/). |
```